File: riesi.go
package main 
  
import (
 "fmt"
 "strings"
)

func main() { 
    var frog string 
    fmt.Scanln(&frog)
    frog = strings.ToLower(frog)
    
    if frog == "riesi" {
        fmt.Print(frog)
    } else {
        fmt.Println("not riesi")
    }
}

File: riesi.dart
import 'dart:io';

void main() {
  String frog = stdin.readLineSync();
  String ribbit = frog.toLowerCase();
  
  if (ribbit == "riesi") {
    stdout.writeln(ribbit);
  } else {
    stdout.writeln("not riesi");
  }
}
